docs/coordinates/velocities.rst

```diff
@@ -417,8 +417,11 @@ radial velocity of 10 km/s:
                radial_velocity=[10]*1000*u.km/u.s,
                location=location, obstime=time)
     gcrs = aa.transform_to(GCRS(obstime=time))
-    plt.plot_date(time.plot_date, gcrs.radial_velocity.to(u.km/u.s))
```

Comment:
How about we replace this with the new way of doing dates, which is to pass in a `datetime`, i.e.,
```
plt.plot(time.datetime, gcrs.radial_velocity.to(u.km/u.s))
plt.ylabel('RV [km/s]')
```